src/clr-angular/utils/_theme.dark.clarity.scss

```diff
@@ -341,6 +341,9 @@ $clr-datagrid-row-hover: #29414e;
 $clr-datagrid-row-selected: $clr-white;
 $clr-datagrid-popover-bg-color: #22343c;
 $clr-datagrid-action-toggle: $clr-datagrid-font-color;
+$clr-datagrid-pagination-btn-color: #e9ecef;
```

Comment:
I know we have some special colors in our dark theme but is `#e9ecef` one of them?

Reply:
It looks like it. We should consider using some sort of SASS variable for it, though. It shows up in this file 9 times already.

Reply:
That can be said about a lot of colors in this file though. Would it be ok to punt this by creating a generic issue to create SASS variables for dark theme colors?

Reply:
Absolutely.

Reply:
Created an issue for this: https://github.com/vmware/clarity/issues/2783